SPECS-EXTENDED/spirv-tools/spirv-tools.spec

```diff
@@ -1,27 +1,20 @@
-%global commit 67f4838659f475d618c120e13d1a196d7e00ba4b
-%global shortcommit %(c=%{commit}; echo ${c:0:7})
-%global commit_date 20200421
-%global gitrel .%{commit_date}.git%{shortcommit}
-
 Name:           spirv-tools
-Version:        2019.5
-Release:        3%{?dist}
+Version:        2022.1
+Release:        1%{?dist}
 Summary:        API and commands for processing SPIR-V modules
 
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/KhronosGroup/SPIRV-Tools
-Source0:        %url/archive/%{commit}.tar.gz#/%{name}-%{commit}.tar.gz
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/SPIRV-Tools-%{version}.tar.gz
+
+Patch0: fix-gcc12-build.patch
```

Comment:
Nit: formatting

Reply:
If I'm just updating a `SPECS-EXTENDED` spec to make it build, I don't really do any clean-up - that's to be done in bulk later or when the spec is migrated to `SPECS`. Right now, I'm usually trying to limit the number of modified lines to show only the important changes. The story is different for `SPECS`, where we try to keep or bring them to a cleaner state, since we consider them something that's more "core" and is a lot more likely to stay with Mariner.